### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by coreice** [Neo11/ChitChat#21]
Will there be a possibility to add translations? Not really a necessary feature, but could be fun.

**Comment by c-o-m-m-a-n-d-e-r** [comment]
Already a Task of mine ;-)

**Comment by coreice** [comment]
Great :D

**Comment by c-o-m-m-a-n-d-e-r** [comment]
translation is limited to the native app parts (like quick reply text etc) 

the whatsapp parts are translated by whatsapp self

**Comment by coreice** [comment]
Yeah and it's already translated, in my case in Dutch. Only the stuff like the menu bar etc are not translated. That's the reason I was wondering about translations of the native parts :).

**Comment by c-o-m-m-a-n-d-e-r** [comment]
yep the native parts will be translated soon ...
i was born near dutch border but i think i need help for the dutch translation ;-)

**Comment by VagishVela** [comment]
@c-o-m-m-a-n-d-e-r  I don't mind helping out with translations.
